caffe2/core/macros.h.in

```diff
@@ -67,4 +67,6 @@
   {"USE_ITT", "${CAFFE2_USE_ITT}"}, \
   {"USE_ROCM_KERNEL_ASSERT", "${USE_ROCM_KERNEL_ASSERT}"}, \
   {"USE_CUSPARSELT", "${USE_CUSPARSELT}"}, \
+  {"USE_XPU", "${USE_XPU}"}, \
+  {"USE_XCCL", "${USE_XCCL}"}, \
```

Comment:
`USE_XCCL` will not work as expected because it is defined in `torch-xpu-ops` and may be not updated.

Reply:
This PR depends on https://github.com/intel/torch-xpu-ops/pull/1377.